ui/main/src/app/modules/admin/components/cell-renderers/signal-mappings-cell-renderer.component.html

```diff
@@ -0,0 +1,26 @@
+<!-- Copyright (c) 2024, RTE (http://www.rte-france.com)                   -->
+<!-- See AUTHORS.txt                                                       -->
+<!-- This Source Code Form is subject to the terms of the Mozilla Public   -->
+<!-- License, v. 2.0. If a copy of the MPL was not distributed with this   -->
+<!-- file, You can obtain one at http://mozilla.org/MPL/2.0/.              -->
+<!-- SPDX-License-Identifier: MPL-2.0                                      -->
+<!-- This file is part of the OperatorFabric project.                      -->
+
+<div class="container" style="margin-top: 6px;text-align: center;">
+        <div class="row" style="border-style: solid; border-width: 1px 1px 0px 1px">
+                <div class="col-sm" style="border-style: solid; border-width: 0px 1px 0px 0px" translate>shared.severity.alarm</div>
+                <div class="col-sm">{{supportedSignals['ALARM']}}</div>
+        </div>
+        <div class="row" style="border-style: solid; border-width: 1px 1px 0px 1px">
+                <div class="col-sm" style="border-style: solid; border-width: 0px 1px 0px 0px" translate>shared.severity.action</div>
+                <div class="col-sm">{{supportedSignals['ACTION']}}</div>
+        </div>
+        <div class="row" style="border-style: solid; border-width: 1px 1px 0px 1px">
+                <div class="col-sm" style="border-style: solid; border-width: 0px 1px 0px 0px" translate>shared.severity.information</div>
+                <div class="col-sm">{{supportedSignals['INFORMATION']}}</div>
+        </div>
+        <div class="row" style="border-style: solid; border-width: 1px 1px 1px 1px">
+                <div class="col-sm" style="border-style: solid; border-width: 0px 1px 0px 0px" translate>shared.severity.compliant</div>
```

Comment:
add scss file to style the table and/or try to implement the opfab-table style ?